### DISCUSSION TITLE
Langsmith Not Logging Multiple LLM Calls?

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#1822] [Q&A]
In the Marketplace example "Prompt Chaining", each query to the Chatflow causes it to issue a chain of two prompts to the LLM - good. But it seems that Langsmith, Langfuse and Lunary only log one LLM call, not two. Is this by design, a bug or am I using the monitoring incorrectly?

**Comment by vinodkiran** [comment]
This is known issue, we are in the process of raising a PR with the fix.

**Comment by joffreygonin** [comment]
Hello, would it be possible to communicate when (or for which version) it is intended to be fixed ? (very important feature).
Thank you.
